sample-input/gradients/cube.py

```diff
@@ -0,0 +1,80 @@
+import numpy
+from openmoc import *
+import openmoc.log as log
```

Comment:
The ``openmoc.log``, ``openmoc.plotter`` and ``openmoc.options`` submodules are not used in this file